### PR TITLE
Add enhanced training script

### PR DESCRIPTION
**Pull request by djalova**
Address #8

### CODE REVIEW

File: train.sh [hunk 1]
```diff
@@ -1,16 +1,74 @@
...
+  echo 'Uploading training data to bucket'
+  aws --endpoint-url=http://s3-api.us-geo.objectstorage.softlayer.net s3 cp $DATA_FILE s3://$TRAINING_BUCKET/$DATA_FILE
+
+  CREATE_SUCCESS=1
+  while [[ $CREATE_SUCCESS ]]; do
+    echo 'Enter a results bucket name'
+    read RESULTS_BUCKET
+    RESULTS_BUCKET=$(echo $RESULTS_BUCKET|tr -d '\n')
+    CREATE_SUCCESS=$(make_bucket $RESULTS_BUCKET)
+  donegit s
```
Comment (ptitzler) [inline train.sh]: Breaks the script.

**Review by ptitzler — COMMENTED** [review]
I was able to train the model with the latest fix.

**Comment by ptitzler** [comment]
Since you are already touching the repo, could you please also update the readme to: 

- use `ibmcloud` (new name)  instead of  `bx` (old name)
- fix the formatting issue (last bullet in section "Setup an IBM Cloud Object Storage (COS) account")

**Comment by djalova** [comment]
Sure, I can update those. I'm not sure what the formatting issue is in the last bullet though.

**Comment by ptitzler** [comment]
screen shot 2018-10-19 at 2 19 24 pm
" - Export ..." looks like it was supposed to be a new item on the bulleted list?

**Comment by djalova** [comment]
Oh thanks for catching. The markdown preview on my editor displays it fine. I'll look into it.

File: README.md [hunk 2]
```diff
@@ -68,7 +68,7 @@ theta = tf.Variable(initial_value=identity)
 ### Setup an IBM Cloud Object Storage (COS) account
 - Create an IBM Cloud Object Storage account if you don't have one (https://www.ibm.com/cloud/storage)
 - Create credentials for either reading and writing or just reading
-	- From the bluemix console page (https://console.bluemix.net/dashboard/apps/), choose `Cloud Object Storage`
+	- From the bluemix console page (https://console.bluemix.net/dashboard/apps/) choose `Cloud Object Storage`
```
Comment (ptitzler) [inline README.md:71]: > From the bluemix console
From the IBM Cloud console

Comment (ptitzler) [reply README.md:71]: Looks like you missed one instance of `bx` in the scoring section. If you could also add a `$` prefix to each code snippet that people need to run in a terminal window to clearly distinguish it from output:

> bx ml score scoring-payload.json

$ ibmcloud ml score scoring-payload.json

**Review by ptitzler — CHANGES_REQUESTED** [review]
Having read the Watson Studio documentation I believe finally understand why the model code was ZIPed up in the first place. We'll need to discuss how to proceed.

**Pull request closed without merging**